integration-tests/regression.test.js

```diff
@@ -221,4 +221,44 @@ describe('Tram-One - regressions', () => {
 		// cleanup - remove app
 		appContainer.remove()
 	})
+
+	it('should not error when reseting focus if the number of elements changed', async () => {
```

Comment:
the test app needed to be updated, and actually caused a previous test to fail without the fix, but it still feels worthwhile to give this it's own test, so that if they ever get refactored, we don't forget about it.